mithril-test-lab/mithril-end-to-end/src/main.rs

```diff
@@ -103,12 +107,20 @@ pub struct Args {
     #[clap(long)]
     run_only: bool,
 
-    /// Enable P2P network mode
+    /// Use Mithril relays
     #[clap(long)]
-    use_p2p_network: bool,
+    use_relays: bool,
+
+    /// Signer registration relay mode (used only when 'use_relays' is set, can be 'passthrough' or 'p2p')
+    #[clap(long, default_value = "passthrough")]
+    relay_signer_registration_mode: String,
```

Comment:
Wouldn't it be more consistent to use an enum instead, just like you did for the relay?
```suggestion
    #[clap(long, default_value_t = SignerRelayMode::Passthrough)]
    relay_signer_registration_mode: SignerRelayMode,
```

Reply:
I think a constant should be fine, adding a full enum would be more boilerplate than anything and would duplicate the relay code.

Reply:
Sounds good to me.

Reply:
Yes, this would add lot of boilerplate, also we have used strings for all other enum equivalents in the e2e tests (such as eras)